modules/richaudienceBidAdapter.js

```diff
@@ -103,10 +103,12 @@ export const spec = {
       if (response.media_type === 'video') {
         bidResponse.vastXml = response.vastXML;
         try {
-          if (JSON.parse(bidRequest.data).videoData.format == 'outstream') {
-            bidResponse.renderer = Renderer.install({
-              url: 'https://cdn3.richaudience.com/prebidVideo/player.js'
-            });
+          if (bidResponse.vastXml != null) {
+            if (JSON.parse(bidRequest.data).videoData.format == 'outstream' || JSON.parse(bidRequest.data).videoData.format == 'banner') {
+              bidResponse.renderer = Renderer.install({
+                url: 'https://cdn3.richaudience.com/prebidVideo/player.js'
```

Comment:
It will be helpful to publishers configuring Outstream in different variations for your adapter to pass additional information to Renderer.install like 'id', 'config', 'loaded', and adUnitCode

Reply:
From Rich Audience, we don’t need additional info since our player does render the video correctly already with the current params. Do you think that these params would be mandatory?

Reply:
The additional information helps Prebid core to parse renderers in the auction. There are many places a renderer can be chosen. If you look to other modules it shouldn't be hard to populate these values.
id = bidrequest.id
adunitcode = bidrequest.adunitcode
Defining loaded = false is important since you are loading an external script for your renderer.
config = 'mediaTypes.video' or 'oustream'